CDCgov/NEDSS-Modernization
# Page info panel metadata fix

## Description

In the Classic NBS, the Excel File gets downloaded when the user taps on the “Page Metadata” button. The code change in Modern followed the same interaction, and the same data and column will be generated. 

## Tickets

https://cdc-nbs.atlassian.net/browse/CNFT2-1127

## Steps to verify

1. From Classic, click on button "Page Metadata" in page "PreviewPage" to get the CSV.
2. From Modern API, use this endpoint to generate the same CSV: http://localhost:9095/nbs/page-builder/swagger-ui/#/page-controller/downloadPageMetadataUsingGET
3. The two CSV's should match.
